src/sql/azdata.proposed.d.ts

```diff
@@ -894,6 +895,20 @@ declare module 'azdata' {
 		Informational = 'Informational'
 	}
 
+	export enum CardType {
+		Image = 'Image'
+	}
+
+	export interface CardImage {
+		path: URI,
+		size?: string, // auto|length|cover|contain|initial|inherit
```

Comment:
These two types should ideally be defined as actual types, not just plain strings.

Reply:
ping

Reply:
This type has been removed with the move to use the icon for the image.